docs/user/canvas.asciidoc

```diff
@@ -18,16 +18,170 @@ With Canvas, you can:
+
+. Open the menu, then go to *Canvas*.
+
+. On the *Canvas workpads* view, select *Templates*.
+
+. Click the preconfigured template that you want to use.
+
+. Add your own *Name* to the workpad.
+
+[float]
+[[import-existing-workpads]]
+=== Import existing workpads
+
+When you want to use a workpad that someone else has already started, import the JSON file into Canvas.
+
+. Open the menu, then go to *Canvas*.
+
+. On the *Canvas workpads* view, click and drag the file to the *Import workpad JSON file* field.
+
+[float]
+[[use-sample-data-workpads]]
+=== Use sample data workpads
+
+Each of the sample data sets comes with a Canvas workpad that you can use for your own workpad inspiration.
+
+. Add a {kibana-ref}/add-sample-data.html[sample data set].
+
+. On the *Add Data* page, click *View data*, then select *Canvas*.
+
+[float]
+[[add-canvas-elements]]
+== Add elements
+
+Create a story about your data by adding elements to your workpad that include images, text, charts, and more. You can create your own elements and connect them to your data sources, add saved objects, and add your own images.
+
+[float]
+[[create-elements]]
+=== Create elements
+
+Choose the type of element you want to use, then use the preconfigured demo data to familiarize yourself with the element. When you're ready, connect the element to your own data. By default, most of the elements you create use 
+demo data until you change the data source. The demo data includes a small data set that you can use to experiment with your element.
+
+To begin, click *Add element*, then select the element you want to use.
+
+[role="screenshot"]
+image::images/canvas-element-select.gif[Canvas elements]
+
+When you're ready to connect the element to your data, select *Data*, then select one of the following data sources:
+
+* *{es} SQL* &mdash; Access your data in {es} using SQL syntax. For information about SQL syntax, refer to {ref}/sql-spec.html[SQL language].
+
+* *{es} documents* &mdash; Access your data in {es} without using aggregations. To use, select an index and fields, and optionally enter a query using the <<lucene-query,Lucene Query Syntax>>. Use the *{es} documents* data source when you have low volume datasets, to view raw documents, or to plot exact, non-aggregated values on a chart.
+
+* *Timelion* &mdash; Access your time series data using <<timelion,Timelion>> queries. To use Timelion queries, you can enter a query using the <<lucene-query,Lucene Query Syntax>>.
+
+Each element can display a different data source, and pages and workpads often contain multiple data sources.
+
+When you're ready to save your element, select the element, then click *Edit > Save as new element*.
+
+[role="screenshot"]
+image::images/canvas_save_element.png[]
+
+To save a group of elements, press and hold Shift, select the elements you want to save, then click *Edit > Save as new element*.
+
+Elements are saved in *Add element > My elements*.
```

Comment:
Instead of "Elements are saved" how about "To view your saved elements, click ..."